package.json

```diff
@@ -22,7 +22,7 @@
     "lodash.assign": "^4.0.8"
   },
   "peerDependencies": {
-    "react": "^0.14.8"
+    "react": "^0.14.8 | ~15.0.0"
```

Comment:
How about `>=0.14.8`? From http://semver.npmjs.com/ this looks like:
<img width="1105" alt="screen shot 2016-06-17 at 10 51 00 am" src="https://cloud.githubusercontent.com/assets/1270189/16159766/6eceec04-3479-11e6-98a7-50ceff46af42.png">